### PR TITLE
Add Sentry error tracking

### PR DESCRIPTION
**Pull request by rshipp**
Task: https://app.asana.com/0/1136372351092870/1137743756490017

Summary:

* Add the Sentry gem; needs Heroku env variable `SENTRY_DSN`.

### CODE REVIEW

**Comment by rshipp** [comment]
alright, got rvm working locally and pushed up the lockfile

File: Gemfile.lock [hunk 1]
```diff
@@ -341,11 +343,15 @@ DEPENDENCIES
...
   turbolinks
   uglifier (>= 1.3.0)
   webmock
 
+RUBY VERSION
+   ruby 2.2.3p173
+
 BUNDLED WITH
-   1.10.6
+   1.17.3
```
Comment (StetsenTech) [inline Gemfile.lock]: I think this needs to stay `1.10.6`

File: Gemfile.lock [hunk 2]
```diff
@@ -341,11 +343,15 @@ DEPENDENCIES
...
   sdoc
+  sentry-raven
   therubyracer
   to_xls-rails
   turbolinks
   uglifier (>= 1.3.0)
   webmock
 
+RUBY VERSION
+   ruby 2.2.3p173
```
Comment (StetsenTech) [inline Gemfile.lock]: I don't think we want this in here either.

File: Gemfile.lock (hunk 1 above)
Comment (hcli-work) [reply Gemfile.lock]: good catch

**Comment by rshipp** [comment]
@sadleb I'm not sure what to do about the version changes @StetsenTech pointed out above. I can reset those changes manually in the Gemfile.lock but that seems messy.

File: Gemfile.lock (hunk 1 above)
Comment (sadleb) [reply Gemfile.lock]: Ditto, let's make sure we are using the same version of bundler that we have been. Heroku choose the version of bundler to use based on this line. See: https://devcenter.heroku.com/articles/ruby-support#libraries

**Comment by sadleb** [comment]
> @sadleb I'm not sure what to do about the version changes @StetsenTech pointed out above. I can reset those changes manually in the Gemfile.lock but that seems messy.

I have bundler 1.10.6 installed under my ruby-2.2.3 rvm env and tried a `bundle install`. You can either get the correct version of bundler installed and run it or just remove the `RUBY VERSION
   ruby 2.2.3p173` and revert the `BUNDLED WITH
   1.17.3` line since that's the only difference.

**Comment by rshipp** [comment]
👍 extraneous versions removed

**Pull request merged**